kwok/main.go

```diff
@@ -40,5 +40,5 @@ func main() {
 			state.NewCluster(op.Clock, op.GetClient(), cloudProvider),
 			op.EventRecorder,
 			cloudProvider,
-		)...).Start(ctx)
+		)...).Start(ctx, cloudProvider)
```

Comment:
Need to make this change here?

Reply:
Yes, since the Kwok provider will need to be updated as the operator in upstream is updated, no?